src/pages/GithubApp/Components/Container.jsx

```diff
@@ -0,0 +1,228 @@
+import React from 'react';
+import octocat from '../../../assets/github-app-img/octocat.png';
+import { useState } from 'react';
+import search from '../../../assets/github-app-img/icon-search.svg';
+import dayjs from 'dayjs';
+import useDarkMode from '../../../hook/useDarkMode';
+
+function Container() {
+  const [show, setShow] = useState('');
+  const [user, setUser] = useState('');
+  const [githubData, setGithubData] = useState({
+    name: 'The Octocat',
+    avatar_url: octocat,
+    join_date: '25 Jan 2011',
+    username: 'octocat',
+    bio: 'This user has no bio',
+    followers: '3938',
+    following: '9',
+    public_repos: '8',
+    location: 'San Francisco',
+    blog: 'https://github.blog',
+    twitter_username: 'Not Available',
+    company: '@github',
+  });
+  useDarkMode();
+
+  const handleShowClick = async () => {
+    const response = await fetch(`https://api.github.com/users/${user}`);
```

Comment:
Try catch the fetch for error handling - even if you just log the error to console.